AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/SystemKey/Handler.dfy

```diff
@@ -0,0 +1,374 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+include "../../Model/AwsCryptographyKeyStoreAdminTypes.dfy"
+include "../KmsUtils.dfy"
+include "ContentHandler.dfy"
+
+/* Public methods for Signing and Verifying Mutation Items */
+module {:options "/functionSyntax:4" } SystemKey.Handler {
+  import opened Wrappers
+  import opened StandardLibrary.UInt
+  import opened StandardLibrary.NeedError
+  import UTF8
+  import KMS = Com.Amazonaws.Kms
+  import Types = AwsCryptographyKeyStoreAdminTypes
+  import KSTypes = AwsCryptographyKeyStoreAdminTypes.AwsCryptographyKeyStoreTypes
+  import KmsUtils
+  import AtomicPrimitives
+  import ContentHandler // = SystemKey.ContentHandler
+  import Structure
+  import MPL = MaterialProviders
+
+  // type SystemKeyError = e: Types.Error | e.MutationVerificationException? witness *
+
+  // https://cyberchef.infosec.amazon.dev/#recipe=Encode_text('UTF-8%20(65001)')To_Decimal('Comma',false)&input=Y3JlYXRlLXRpbWU&oenc=65001&oeol=CR
```

Comment:
let's look through all the constants we have introduced and make sure we are following a common pattern (camelCase, PascalCase, etc..)